desktop/install/debian.md

```diff
@@ -49,10 +49,12 @@ Recommended approach to install Docker Desktop on Debian:
 
 1. Set up [Docker's package repository](../../engine/install/debian.md#set-up-the-repository). 
 
-2. Download latest DEB package from the [release](../release-notes.md) page.
+2. Download latest DEB package:
```

Comment:
Sorry @p1-0tr , that was a bad idea of mine - this doesn't look great. Let's just keep it as it was with the links

Reply:
Done :) I made them direct download links, but inline